exporter/kafkaexporter/README.md

```diff
@@ -0,0 +1,24 @@
+# Kafka Exporter
+
+Kafka exporter exports traces to Kafka.
+
+The following settings are required:
+- `protocol_version` (no default): Kafka protocol version e.g. 2.0.0
```

Comment:
Please mention in the readme what is the proto that we write to the queue.

Reply:
There will be a follow-up PR to make that configurable. 

Since we already import Jaeger and Zipkin we should make the encoding configurable. This will allow folks easy migration.